src/main/java/org/simdjson/SimdJsonParser.java

```diff
@@ -24,7 +26,12 @@ public SimdJsonParser(int capacity, int maxDepth) {
         paddedBuffer = new byte[capacity];
         indexer = new StructuralIndexer(bitIndexes);
     }
-
+    public BitIndexes buildBitIndex (byte[] buffer, int len) {
```

Comment:
Should we expose this method publicly or should it only be used by the library itself?
```suggestion
    BitIndexes buildBitIndex (byte[] buffer, int len) {
```